scraper.rb

```diff
@@ -2,239 +2,38 @@
 
 require 'dotenv'
 Dotenv.load
-require 'octokit'
-require 'scraperwiki'
-require 'mechanize'
-require 'rest-client'
-require 'json'
-require 'active_support/all'
-
-# Duck punches
-class Numeric
-  def percent_of(n)
-    to_f / n.to_f * 100.0
-  end
-end
-
-# PlanningAlerts contributor stats from GitHub
-class GitHub
-  class << self
-    def commits_text(period:)
-      puts 'Collect information from GitHub'
-      if commits_count(period: period).zero?
-        nil
-      else
-        "You shipped #{commits_count(period: period)} commits in the same period."
+require_relative 'lib/github'
+require_relative 'lib/planningalerts'
+require_relative 'lib/righttoknow'
+require_relative 'lib/runner'
+
+module Jacaranda
+  # The runner for PlanningAlerts
+  class PlanningAlerts < Runner
```

Comment:
@auxesis I found this way of adding runners a bit confusing with developers having to add methods here and then also to library files. The way Jacaranda selects which scrapers to run seems a bit automagical :)

What do you think of this alternate approach for adding runners:

1. add a file for your runner to ./lib/runners/ . It should define your runner as a Class that inherits from Runner.
2. Your class should define a `build` method that contains the text you want jacaranda to post to slack.
3. In scraper.rb add the Class name of your runner to the list in `Jacaranda.run(runners: ["PlanningAlerts", "RightToKnow", "GotGastro"])`.
4. That's it.

The advantage of that is that you only add one file and it's transparent how Jacaranda is instructed what to run and intuitive how to change it.